samples/cpp/benchmark_app/main.cpp

```diff
@@ -1132,13 +1132,6 @@ int main(int argc, char* argv[]) {
             if (FLAGS_api == "sync") {
                 inferRequest->infer();
             } else {
-                // As the inference request is currently idle, the wait() adds no
-                // additional overhead (and should return immediately). The primary
-                // reason for calling the method is exception checking/re-throwing.
-                // Callback, that governs the actual execution can handle errors as
-                // well, but as it uses just error codes it has no details like ‘what()’
-                // method of `std::exception` So, rechecking for any exceptions here.
-                inferRequest->wait();
```

Comment:
Why it is removed?

Reply:
The explanation for calling `inferRequest->wait(;` claims 
> callback uses just error codes it has no details like ‘what()’

This is no longer true. Callback provides an exception now.